ogticrd/covid-safe-paths
# deleting an unused function and refactor positiveOnboarding dialog

#### Description:


taking out share location dialog from positive onboarding file and deleting an unused function also fix a problem with a dependency form react-native-pdf
#### Linked issues:


Fix: [245](https://trello.com/c/XNlQq6Tr/245-as-a-developer-i-want-to-refactor-the-code-so-that-i-can-find-duplicate-code-3)

#### How to test:


- `git pull && git checkout Fix/245-Deleting-unused-function`
- report yourself as positive covid user and all the flow should work as was expected
